formats_err/imports_enum_leaking.ksy

```diff
@@ -0,0 +1,25 @@
+# imports/enum_two.ksy: /seq/0/enum:
+# 	error: unable to find type 'enum_one', searching from 'enum_two'
+#
+# imports/enum_two.ksy: /instances/instance_one/enum:
+# 	error: unable to find type 'enum_one', searching from 'enum_two'
+#
```

Comment:
Strange, but the second error never reported in this test, even when I apply https://github.com/kaitai-io/kaitai_struct_compiler/pull/313, so all errors should be reported. Don't figure yet, why